### PR TITLE
Committing Solutions/VPLS-view on behalf of Devang Patel

### PR DESCRIPTION
**Pull request by ipmonk**

### CODE REVIEW

**Comment by ayushgithub** [comment]
jenkins please retry a build

**Comment by ayushgithub** [comment]
jenkins please retry a build

File: juniper_official/Solutions/VPLS-view/vpls.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (ayushgithub) [inline juniper_official/Solutions/VPLS-view/vpls.yml]: 472
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/VPLS-view/vpls.yml
  7:1       error    syntax error: could not find expected ':'

Comment (ayushgithub) [inline juniper_official/Solutions/VPLS-view/vpls.yml]: 478
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/VPLS-view/vpls.yml
  7:1       error    syntax error: could not find expected ':'

**Comment by ayushgithub** [comment]
jenkins please retry a build

File: juniper_official/Solutions/VPLS-view/vpls.yml (hunk 1 above)
Comment (ayushgithub) [inline juniper_official/Solutions/VPLS-view/vpls.yml]: 500
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/VPLS-view/vpls.yml
  9:1       error    syntax error: could not find expected ':'

**Comment by ayushgithub** [comment]
jenkins please retry a build

File: juniper_official/Solutions/VPLS-view/vpls-view.playbook [hunk 2]
```diff
@@ -0,0 +1,5 @@
+iceberg {
+    playbook vpls-view {
```
Comment (gaddevijay) [inline juniper_official/Solutions/VPLS-view/vpls-view.playbook]: Could you please add comments detailing about the playbook functionality. Please look at below example.

https://github.com/Juniper/healthbot-rules/blob/master/juniper_official/Linecard/linecard-kpis.playbook

This comment is valid for rules as well

**Pull request merged**